Library/Homebrew/extend/os/ENV/super.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/super"
```

Comment:
I agree that `extend/os/extend` feels weird. But I think this is trumped by a more mechanical mapping where it is guaranteed that the platform-specific bits of `<path>` always live in `extend/os/<path>` (the dispatcher) and `extend/os/<platform>/<path>` (the implementation).